js/deribit.js

```diff
@@ -1790,16 +1790,18 @@ module.exports = class deribit extends Exchange {
         for (let i = 0; i < positions.length; i++) {
             result.push (this.parsePosition (positions[i]));
         }
-        // todo unify parsePositions
         return result;
     }
 
-    async fetchPositions (symbols = undefined, params = {}) {
+    async fetchPositions (code = undefined, params = {}) {
```

Comment:
We can't really change the signature of fetchPositions. I suggest that we do something like this: https://github.com/ccxt/ccxt/blob/master/js/deribit.js#L562

Reply:
> We can't really change the signature of fetchPositions, it must remain unified across the exchanges. I suggest that we do something like this: https://github.com/ccxt/ccxt/blob/master/js/deribit.js#L562

Okay I'm a bit unsure of how codeFromOptions works, it seems to always return 'BTC'

Reply:
Can I just change code to symbols so that it looks the same as other unified methods? I think that code and symbols are the same thing, I'm not sure that I need to use codeFromOptions

Reply:
`codeFromOptions` works almost the same as `handleMarketTypeAndParams`.

If the user specified a code parameter, that is taken as the currency code. If not, then a default currency code is checked in the exchange options for this or that method (fetchBalance, or, in this case, fetchPositions). 

We should not really mix currency codes with market symbols, they're not interchangeable. Yet, in this case, I would suggest to support both a code in the `symbol` argument and `codeFromOptions` – it's not that hard to write out the sequence of checks for that. Let me know if you have questions or issues on it.